server/server.go

```diff
@@ -1400,3 +1400,7 @@ func (s *Server) SaveTTLConfig(data map[string]interface{}, ttl time.Duration) e
 	}
 	return nil
 }
+
+func (s *Server) SplitAndScatterRegions(context context.Context, r *pdpb.SplitAndScatterRegionsRequest) (*pdpb.SplitAndScatterRegionsResponse, error) {
+	return nil, nil
+}
```

Comment:
Why add this?

Reply:
for pass compile, kvproto add this method